docs/admin/troubleshooting/docker-jcmd.rst

```diff
@@ -18,6 +18,14 @@ bit differently. This document explains why the 'usual' way to run ``jcmd``
 does not work and how to solve it. It does not, however, explain how to analyze
 the output (since that is identical to non-containerized applications)!
 
+.. note::
+
+    Other than telling readers about "You can find a lot of troubleshooting
+    guides out there...", we should provide canonical instructions, optionally
+    supported by corresponding tooling, and tailored to typical environments
+    and scenarios CrateDB users find themselves in.
```

Comment:
On the `index.md` page, I've now included references to those.

- https://www.baeldung.com/running-jvm-diagnose
- https://www.baeldung.com/java-flight-recorder-monitoring

Do you think they can count as canonical resources on this matter? Please let us know if you think we should refer to / also includes links to other resources. For example, what about jcmd/JFR integration for popular IDEs like IntelliJ or VSCode?

/cc @matriv, @BaurzhanSakhariev

Reply:
- https://github.com/crate/support/issues/366